sequencer/README.md

```diff
@@ -1,5 +1,7 @@
 # Sequencer for SYB
 
+We are developing a zk-rollup sequencer designed to compute a uniqueness score for each user account based on the vouches made between accounts. This score will reflect the interactions and endorsements that one account provides to another.The sequencer will serve as a backend middleware, facilitating communication between the smart contract and the Circuit, which is responsible for calculating the uniqueness index. By utilizing a zk-rollup architecture, we aim to significantly reduce the computational costs associated with calculating scores directly through smart contracts, which would otherwise be prohibitively high. This approach ensures efficiency and scalability while maintaining the integrity of our computations.
```

Comment:
Just a minor suggestion, how about using a passive voice and removing the subject? so instead of `We are developing a zk-rollup sequencer`, `SYB sequencer is a zk-rollup sequencer designed to...`